apps/settings/js/modules/date_time.js

```diff
@@ -193,9 +193,7 @@ define(function(require) {
   DateTime.prototype.handleEvent = function(evt) {
     switch (evt.type) {
       case 'moztimechange':
-        navigator.mozL10n.ready((function _updateTime() {
-          this._autoUpdateDateTime();
-        }).bind(this));
+        this._autoUpdateDateTime();
```

Comment:
Because you don't need them anymore! The only reason we had them was for synchronous `mozL10n.get` that we're obsoleting and I'm removing all calls to it in this patch.

The `mozL10n.ready` serves one more purpose - it refires on language change, but in this case it was a bug because it was firing new `mozL10n.ready` on every moztimechange, so after a few, the callback was fired multiple times on languagechange.

With this particular removal, the autoUpdateDateTime will happen only when moztimechange happens.